src/lib/utils.c

```diff
@@ -39,15 +39,15 @@ twist aes256_gcm_decrypt(const twist key, const twist objauth) {
 
     char *tag = strchr(objcopy, ':');
     if (!tag) {
-        LOGE("Could not find :");
+        LOGE("Could not find : to split tag");
         goto out;
     }
     *tag = '\0';
     tag++;
 
     char *ctext = strchr(tag, ':');
     if (!ctext) {
-        LOGE("Could not find :");
+        LOGE("Could not find : to split ctext");
```

Comment:
split to different patch for both changes in utils.c. This is just error reporting enhancement.